app/views/locales/index.html.erb

```diff
@@ -3,7 +3,7 @@
 <!--/descriptions-->
 
 <!--main content-->
-<% if   logged_in_as_admin? || permit?("translation_admin")  %>
+<% if   logged_in_as_admin? || permit?("translation_admin", :get_user_method => 'current_user')  %>
```

Comment:
Hound doesn't run on .html.erb files, so it didn't mention that you should probably change to the `get_user_method: 'current_user'` syntax here as well. If you wanted to fix the extra spaces between `if` and `logged_in_as_admin?` while you were editing this, that would be great.